lib/app_web/controllers/profile_controller.ex

```diff
@@ -0,0 +1,41 @@
+defmodule AppWeb.ProfileController do
+  use AppWeb, :controller
+  alias App.Person
+  plug :permission_profile when action in [:edit, :update]
+
+  def edit(conn, %{"person_id" => person_id}) do
+    profile = Person.get_person!(person_id)
+    changeset = Person.changeset(profile)
+    render(conn, "edit.html", profile: profile, changeset: changeset)
+  end
+
+  def update(
+        conn,
+        %{"person_id" => person_id, "person" => person_params}
+      ) do
+    profile = Person.get_person!(person_id)
+
+    case Person.update_person(profile, person_params) do
+      {:ok, _person} ->
+        conn
+        |> put_flash(:info, "Person updated successfully.")
```

Comment:
Guessing this "Person updated successfully." is auto-generated? 🤖 
Maybe we could change it to "Your profile was updated." or something a bit more friendly? 💭

Reply:
The flash messages are not displayed at the moment.
The function to display them might have been removed previously, I need to have a look at it but we can definitely update the message